video.py

```diff
@@ -57,23 +61,28 @@ def get_segments(manifest_url):
 
     return segment_urls
 
+
 def merge_segments(segment_urls, output_filename):
     print("Download segments")
     for channel, urls in segment_urls.items():
         with open(f"_{channel}.mp4", "wb") as out_file:
             print(channel, ": ", 0, "/", len(urls), sep="", end="", flush=True)
-            for i, url in enumerate(urls):
+            for i, url in enumerate(urls, start=1):
                 res = requests.get(url)
                 out_file.write(res.content)
                 print("\r", channel, ": ", i, "/", len(urls), sep="", end="", flush=True)
             print()
 
     print("Merge using ffmpeg")
-    subprocess.run(["ffmpeg", "-i", "_audio.mp4", "-i", "_video.mp4", "-c", "copy", output_filename])
-    print("Merge complete, you can remove _audio.mp4 and _video.mp4")
+    subprocess.run(["ffmpeg", "-y", "-i", "_audio.mp4", "-i", "_video.mp4", "-c", "copy", output_filename])
+    print("Merge complete, cleaning up")
+    for channel in segment_urls.keys():
```

Comment:
minor nitpick: use `in segment_urls:` instead of `in segment_urs.keys():`

see for example here: https://stackoverflow.com/a/17634287

Reply:
From the Zen of Python:

> Explicit is better than implicit.

;)